src/io.h

```diff
@@ -40,4 +40,27 @@ void set_timer(int ms, timeout_callback_fn_t* cb);
 void clear_timer();
 
 bool device_is_unlocked();
+
+// UX_TICKER_EVENT from Nano X BOLOS SDK overrides our callbacks so we made our own version of it
+// Ticker event interval is assumed to be 100 ms.
+#if defined(TARGET_NANOS)
```

Comment:
Isn't there `UX_ALLOWED` SDK helper compatible both with S and X ?

Reply:
I haven't found any such helper in the SDK

Reply:
Ok. Anyway, let's keep this in the .c file, directly at the point of use

Reply:
done